### DISCUSSION TITLE
is there a way to make the build time as fast as the original MCP was?

### DISCUSSION THREAD

**Discussion by Noodlegamer76** [Hexeption/MCP-Reborn#251] [Q&A]
when I ran the original MCP a while back the game built in only a few seconds, whereas this takes a few minutes

**Comment by TheKodeToad** [comment]
I'd assume it's because the game has got a lot larger! so there probably isn't. Gradle should only build changed files.

**Reply by APie357** [reply]
Gradle sadly builds all the files, not just the ones that changed. I really find that annoying and that honestly needs to change.

**Reply by TheKodeToad** [reply]
I'm pretty sure building only changes files is one of Gradle's advantages. However, it's still not as fast as Eclipse!
